tokio-trace/Cargo.toml

```diff
@@ -21,7 +22,7 @@ categories = ["development-tools::debugging", "asynchronous"]
 keywords = ["logging", "tracing"]
 
 [dependencies]
-tokio-trace-core = "0.2"
+tokio-trace-core = { version = "0.2", path = "tokio-trace-core" }
```

Comment:
this change was for testing, i can take it out if there's some other way to set this up